### PR TITLE
Create Ground Based Sensor Subsystem.md

### PR DESCRIPTION
**Pull request by mtsisk**

### CODE REVIEW

File: Documentation/Signoffs/Ground Based Sensor Subsystem.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+
+The ground based sensor subsystem is to sense vehicles passing above the sensor, allowing for the system to keep a count of the
+number of cars that are in a specified parking lot on the campus of Tennessee Tech.  This system will be placed at parking lots
+that have specified entrances/exits.  By using lots that have this constraint, it will allow the system to keep an accurate count
+of the number of cars in a specific lot.
+
+# Constraints
+| No. | Constraints | Origin |
+| --- | ----------- | ------ |
+| 1.  | Subsystem shall detect vehicles entering and exiting a parking lot. | System Requirements |
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/Ground Based Sensor Subsystem.md]: Not sufficiently precise. vehicle? entering? exiting?

**Review by JesseTNRoberts — COMMENTED** [review]
This device addresses a failure of the camera systems. However, I don't see anything here addressing that. What I mean is how do you know that this will work in contexts which the cameras don't?

These constraints are not sufficient. You need to consider what you expect of this system and what it must do for it to fulfill that role.

File: Documentation/Signoffs/Ground Based Sensor Subsystem.md (hunk 1 above)
Comment (mtsisk) [reply Documentation/Signoffs/Ground Based Sensor Subsystem.md]: I have updated the Constraints section by adding numeric values to Constraint 1.  I have also defined what the term vehicle means in the updated markdown file.

**Review by JesseTNRoberts — COMMENTED** [review]
Shorten the description. Be direct and brief.

Why cars and pickups?

How tall should you expect the trucks to be?

The schematics are confusing. I'm not sure what I'm looking at.

**Comment by mtsisk** [comment]
From the previous comments from the most recent review, I have listed updates to the file:

1. The description has been updated and is shorter than the previous review.  It does contain information as to how it will be more effective than using cameras.
2. The reasons for cars and pickups is explained in the constraints.  An explanation is also given in the Analysis section.
3. Detection height determination is mentioned in the Analysis section.  Though the detection height is quite large, it does take into consideration vehicles that have been raised from the height that was set to when it was brand new from the assembly line.
4. Schematics have been revised and explained in detail in the Analysis section.

**Review by JesseTNRoberts — COMMENTED** [review]
Keep the discussion, but put the constraints into a table.

I am not certain what this is a schematic of. It isn't sufficiently detailed for what you are designing. Also, it seems rather expensive.

**Review by JesseTNRoberts — COMMENTED** [review]
Why is there a 900$ in cord covers?

**Comment by mtsisk** [comment]
For the most recent review request, a list of what has been updated is provided below.

1. Schematic drawing is updated and includes details for both the inductive loop wires, including types of wire, wire size, and distance needed for the loops, and lead-in connections.
2. Schematic drawing also includes cord covers to protect the wire for the inductive loops and lead-in connection wire.
3. Constraints section is updated to confine the updated schematic drawing to the specific measurements, wire sizes, etc.
4. Analysis section is updated to explain the updated schematic drawing.
5. BOM section is updated to reflect the new price of the new cord cover.  The price is decreased by nearly 50% of the previous selection for cord covers.  This results in the lower total price that comes with this subsystem.

**Review by JesseTNRoberts — COMMENTED** [review]
What is this device doing? If it is simple, why don't we build it? I'm not seeing anything complicated here. We have two wires. why are we paying hundreds of dollars?

**Comment by mtsisk** [comment]
From the comments of the most recent review, the explanation of the Diablo DSP-21 Vehicle Detector has been included in the Analysis section.

**Review by JesseTNRoberts — COMMENTED** [review]
We need constraints and analysis addressing the difficult cases. What about when a car enters while another exits? And the rest of the in class questions. I am not yet convinced this will do everything we want. I'm also a bit unsure if this is something that could be easily built rather than bought.

**Comment by mtsisk** [comment]
From the previous review and comments, the difficult cases have been addressing in both the constraints and the analysis.  Also, more explanation for buying the detector unit is given in the analysis section.

**Review by JesseTNRoberts — COMMENTED** [review]
The cord covers you have specked are far too expensive. I don't see road tubes covered like this. Why is this included?

**Comment by mtsisk** [comment]
From the previous comment and conversation in class on Nov. 6, 2023, this system includes a function generator that will be used in replacement for the system.  The protection that will be used for both the loops and the lead-in connections has been updated.  Though the prices remain in the BOM, the total price of the system has been updated.  The costs; however, will be kept in the BOM for future reference with an asterisk that is placed on the parts that will not be added into the total cost.

**Review by JesseTNRoberts — COMMENTED** [review]
Why do we need 3 of the stall mats? Could we do this with one?

**Review by JesseTNRoberts — APPROVED** [review]

**Pull request merged**